modules/nf-core/dysgu/tests/main.nf.test

```diff
@@ -0,0 +1,73 @@
+// TODO nf-core: Once you have added the required tests, please run the following command to build this file:
+// nf-core modules test dysgu
+nextflow_process {
+
+    name "Test Process DYSGU"
+    script "../main.nf"
+    process "DYSGU"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "dysgu"
+
+    // TODO nf-core: Change the test name preferably indicating the test-data and file-format used
+    test("sarscov2 - bam") {
+
+        // TODO nf-core: If you are created a test for a chained module
+        // (the module requires running more than one process to generate the required output)
+        // add the 'setup' method here.
+        // You can find more information about how to use a 'setup' method in the docs (https://nf-co.re/docs/contributing/modules#steps-for-creating-nf-test-for-chained-modules).
+
+        when {
+            process {
+                """
+                // TODO nf-core: define inputs of the process here. Example:
+                
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.test_data['sarscov2']['illumina']['test_paired_end_bam'], checkIfExists: true)
+                ]
```

Comment:
As a heads up: you will need to add all inputs here:

```
tuple val(meta), path(input_bam), path(input_bam_index)
tuple val(meta2), path(reference_fasta), path(reference_fasta.fai)
tuple val(meta3), path(temp_dir)
```

```suggestion
                input[0] = [
                            file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/cram/test.paired_end.recalibrated.sorted.cram', checkIfExists: true),
                            file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/cram/test.paired_end.recalibrated.sorted.cram.crai', checkIfExists: true),
                ]
                input[1] = 
                    [ id:'test'], // meta map
                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/genome.fasta', checkIfExists: true),
                   file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/genome.fasta.fai', checkIfExists: true)
                ]
```

Reply:
> As a heads up: you will need to add all inputs here:
> 
> ```
> tuple val(meta), path(input_bam), path(input_bam_index)
> tuple val(meta2), path(reference_fasta), path(reference_fasta.fai)
> tuple val(meta3), path(temp_dir)
> ```

I am a bit confused and need clarification. Just to confirm, I need to add the same test inputs in both the `main.nf `and `main.nf.test files`, and then run the `nf-core modules test dysgu`?
Thanks.

Reply:
Hi @famosab 
I am a bit confused and need clarification. Just to confirm, I need to add the same test inputs in both the `main.nf `and `main.nf.test files`, and then run the `nf-core modules test dysgu`?
Thanks.